modules/lang-painless/src/test/java/org/elasticsearch/painless/WhenThingsGoWrongTests.java

```diff
@@ -20,59 +20,79 @@
 package org.elasticsearch.painless;
 
 import org.apache.lucene.util.Constants;
+import org.elasticsearch.script.ScriptException;
 
 import java.lang.invoke.WrongMethodTypeException;
 import java.util.Arrays;
 import java.util.Collections;
 
 import static java.util.Collections.emptyMap;
 import static java.util.Collections.singletonMap;
+import static org.hamcrest.Matchers.instanceOf;
 
 public class WhenThingsGoWrongTests extends ScriptTestCase {
     public void testNullPointer() {
         expectScriptThrows(NullPointerException.class, () -> {
             exec("int x = params['missing']; return x;");
         });
-    }
-
-    /** test "line numbers" in the bytecode, which are really 1-based offsets */
-    public void testLineNumbers() {
-        // trigger NPE at line 1 of the script
-        NullPointerException exception = expectScriptThrows(NullPointerException.class, () -> {
-            exec("String x = null; boolean y = x.isEmpty();\n" +
-                 "return y;");
+        expectScriptThrows(NullPointerException.class, () -> {
+            exec("Double.parseDouble(params['missing'])");
         });
-        // null deref at x.isEmpty(), the '.' is offset 30 (+1)
-        assertEquals(30 + 1, exception.getStackTrace()[0].getLineNumber());
```

Comment:
I don't care what we test it with, but it is something we return to users, so it should remain tested.